docs/properties.md

```diff
@@ -115,3 +118,5 @@ Alternatively, you can specify image source URL.
 
 * Path: `properties/{linux|windows}Profile/osImage/url`
 * Value: string
+
+Note that newer VMs, such as `Standard_DC8_v2` sizes, require a `gen2` image.
```

Comment:
Maybe something like 'Note that the _v2 version of the DC series VMs require a gen2 image of the OS.'

Reply:
Done